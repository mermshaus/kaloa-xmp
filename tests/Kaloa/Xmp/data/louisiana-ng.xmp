<?xml version="1.0" encoding="UTF-8"?>
<x:xmpmeta xmlns:x='adobe:ns:meta/' x:xmptk='Image::ExifTool 8.75'>
<!-- Taken from: http://www.defenseimagery.mil/imagery.html#a=search&s=isle&guid=1af3c35aa69ea90db3baf66b39533515cf4bf89d -->
<rdf:RDF xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns#'>

 <rdf:Description rdf:about=''
  xmlns:aux='http://ns.adobe.com/exif/1.0/aux/'>
  <aux:ImageNumber>20134</aux:ImageNumber>
  <aux:Lens>18.0-200.0 mm f/3.5-5.6</aux:Lens>
  <aux:LensInfo>180/10 2000/10 35/10 56/10</aux:LensInfo>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:dc='http://purl.org/dc/elements/1.1/'>
  <dc:creator>
   <rdf:Seq>
    <rdf:li>Sgt. Michael L. Owens</rdf:li>
   </rdf:Seq>
  </dc:creator>
  <dc:description>
   <rdf:Alt>
    <rdf:li xml:lang='x-default'>Soldiers of the Louisiana National Guard’s 527th Engineer Battalion, 225th Engineer Brigade, work to reinforce oil protection barriers and land bridges along Elmer’s Island, La., near Grand Isle and Port Fourchon, La., July 9, 2010.</rdf:li>
   </rdf:Alt>
  </dc:description>
  <dc:subject>
   <rdf:Bag>
    <rdf:li>National Guard</rdf:li>
    <rdf:li>oilspill10</rdf:li>
    <rdf:li>clean-up</rdf:li>
    <rdf:li>Elmer's Island</rdf:li>
   </rdf:Bag>
  </dc:subject>
  <dc:title>
   <rdf:Alt>
    <rdf:li xml:lang='x-default'>100709-A-DS601-134</rdf:li>
   </rdf:Alt>
  </dc:title>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:exif='http://ns.adobe.com/exif/1.0/'>
  <exif:ApertureValue>633985/100000</exif:ApertureValue>
  <exif:ColorSpace>1</exif:ColorSpace>
  <exif:Contrast>2</exif:Contrast>
  <exif:CustomRendered>0</exif:CustomRendered>
  <exif:DateTimeDigitized>2010-07-09T12:19:21.60-04:00</exif:DateTimeDigitized>
  <exif:DateTimeOriginal>2010-07-09T12:19:21.60-04:00</exif:DateTimeOriginal>
  <exif:DigitalZoomRatio>1/1</exif:DigitalZoomRatio>
  <exif:ExifVersion>0221</exif:ExifVersion>
  <exif:ExposureBiasValue>0/6</exif:ExposureBiasValue>
  <exif:ExposureMode>0</exif:ExposureMode>
  <exif:ExposureProgram>0</exif:ExposureProgram>
  <exif:ExposureTime>1/320</exif:ExposureTime>
  <exif:FNumber>9/1</exif:FNumber>
  <exif:FileSource>3</exif:FileSource>
  <exif:Flash rdf:parseType='Resource'>
   <exif:Fired>False</exif:Fired>
   <exif:Function>False</exif:Function>
   <exif:Mode>0</exif:Mode>
   <exif:RedEyeMode>False</exif:RedEyeMode>
   <exif:Return>0</exif:Return>
  </exif:Flash>
  <exif:FocalLength>520/10</exif:FocalLength>
  <exif:FocalLengthIn35mmFilm>78</exif:FocalLengthIn35mmFilm>
  <exif:GainControl>0</exif:GainControl>
  <exif:ISOSpeedRatings>
   <rdf:Seq>
    <rdf:li>160</rdf:li>
   </rdf:Seq>
  </exif:ISOSpeedRatings>
  <exif:LightSource>0</exif:LightSource>
  <exif:MaxApertureValue>45/10</exif:MaxApertureValue>
  <exif:MeteringMode>5</exif:MeteringMode>
  <exif:PixelXDimension>2896</exif:PixelXDimension>
  <exif:PixelYDimension>1944</exif:PixelYDimension>
  <exif:Saturation>0</exif:Saturation>
  <exif:SceneCaptureType>0</exif:SceneCaptureType>
  <exif:SceneType>1</exif:SceneType>
  <exif:SensingMethod>2</exif:SensingMethod>
  <exif:Sharpness>0</exif:Sharpness>
  <exif:ShutterSpeedValue>8321928/1000000</exif:ShutterSpeedValue>
  <exif:SubjectDistanceRange>0</exif:SubjectDistanceRange>
  <exif:WhiteBalance>0</exif:WhiteBalance>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:photoshop='http://ns.adobe.com/photoshop/1.0/'>
  <photoshop:AuthorsPosition>U.S. Army Photographer</photoshop:AuthorsPosition>
  <photoshop:City>PORT FOURCHON</photoshop:City>
  <photoshop:ColorMode>3</photoshop:ColorMode>
  <photoshop:Country>United States</photoshop:Country>
  <photoshop:Credit>U.S. Army</photoshop:Credit>
  <photoshop:Headline>Louisiana Guardsmen reinforce oil protection structures near Grand Isle and Port Fourchon</photoshop:Headline>
  <photoshop:ICCProfile>sRGB IEC61966-2.1</photoshop:ICCProfile>
  <photoshop:Instructions>Released</photoshop:Instructions>
  <photoshop:LegacyIPTCDigest>D40D75887FEFD13C875259708E8A9E23</photoshop:LegacyIPTCDigest>
  <photoshop:Source>Digital</photoshop:Source>
  <photoshop:State>La.</photoshop:State>
  <photoshop:TransmissionReference>Louisiana National Guard</photoshop:TransmissionReference>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:tiff='http://ns.adobe.com/tiff/1.0/'>
  <tiff:BitsPerSample>
   <rdf:Seq>
    <rdf:li>8</rdf:li>
    <rdf:li>8</rdf:li>
    <rdf:li>8</rdf:li>
   </rdf:Seq>
  </tiff:BitsPerSample>
  <tiff:ImageLength>1944</tiff:ImageLength>
  <tiff:ImageWidth>2896</tiff:ImageWidth>
  <tiff:Make>NIKON CORPORATION</tiff:Make>
  <tiff:Model>NIKON D80</tiff:Model>
  <tiff:Orientation>1</tiff:Orientation>
  <tiff:PhotometricInterpretation>2</tiff:PhotometricInterpretation>
  <tiff:ResolutionUnit>2</tiff:ResolutionUnit>
  <tiff:SamplesPerPixel>3</tiff:SamplesPerPixel>
  <tiff:XResolution>300/1</tiff:XResolution>
  <tiff:YResolution>300/1</tiff:YResolution>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:xap='http://ns.adobe.com/xap/1.0/'>
  <xap:CreateDate>2010-07-09T12:19:21.60-04:00</xap:CreateDate>
  <xap:CreatorTool>Ver.1.01</xap:CreatorTool>
  <xap:MetadataDate>2010-07-09T18:44:14-04:00</xap:MetadataDate>
  <xap:ModifyDate>2010-07-09T12:19:21.60-04:00</xap:ModifyDate>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:xapRights='http://ns.adobe.com/xap/1.0/rights/'>
  <xapRights:Marked>False</xapRights:Marked>
 </rdf:Description>

 <rdf:Description rdf:about=''
  xmlns:crs='http://ns.adobe.com/camera-raw-settings/1.0/'>
  <crs:AlreadyApplied>True</crs:AlreadyApplied>
 </rdf:Description>
</rdf:RDF>
</x:xmpmeta>
