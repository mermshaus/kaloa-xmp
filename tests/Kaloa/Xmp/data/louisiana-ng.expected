Xmp.dc.Creator: Sgt. Michael L. Owens
Xmp.dc.Description: Soldiers of the Louisiana National Guard’s 527th Engineer Battalion, 225th Engineer Brigade, work to reinforce oil protection barriers and land bridges along Elmer’s Island, La., near Grand Isle and Port Fourchon, La., July 9, 2010.
Xmp.dc.Subject: National Guard
Xmp.dc.Subject: oilspill10
Xmp.dc.Subject: clean-up
Xmp.dc.Subject: Elmer's Island
Xmp.dc.Title: 100709-A-DS601-134

Xmp.exif.DateTimeOriginal: 2010-07-09T12:19:21.600000-04:00
Xmp.exif.ExifVersion: 0221
Xmp.exif.PixelXDimension: 2896
Xmp.exif.PixelYDimension: 1944
